.travis.yml

```diff
@@ -1,20 +1,45 @@
+sudo: required
+dist: trusty
 language: node_js
-sudo: false
+
 node_js:
-- '5.3'
-branches:
-  except:
-  - /^v[0-9\.]+/
+- '6.9.1'
+- 'stable'
+
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    - google-chrome
+    packages:
+    - google-chrome-stable
+    - g++-4.8
+
+cache:
+  directories:
+  - node_modules
+
 before_install:
-- npm install -g coveralls pr-bumper
-- pr-bumper check
+- npm config set spin false
+- npm install -g coveralls pr-bumper@^1.0.0
```

Comment:
Lets go ahead and make this use the newer version 2:
`- npm install -g coveralls pr-bumper@^2.0.0`

We will need to create the `.pr-bumper.json` file as well. It should have this:
```
{
  "features": {
    "changelog": {
      "enabled": true
    }
  }
}

```